overlay/etc/init.d/k3c-service

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+
+depend() {
+    after network-online
+}
+
+start_pre() {
+    rm -f /tmp/k3c.*
+}
+
+supervisor=supervise-daemon
+name=k3c-service
+command="/usr/bin/k3c"
```

Comment:
At this time I installed the init script, but did not add it to any runlevel so that the admin can start it on demand. 

What are your thoughts @dweomer ?